bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/RuleRegistryImpl.java

```diff
@@ -32,10 +38,14 @@
     private Set<String> disabledRuledSet = new HashSet(0);
     private Logger logger;
     private Storage<Boolean> disabledRulesStorage;
+    private EventPublisher eventPublisher;
+    
+    private static final String SOURCE = RuleRegistryImpl.class.getSimpleName();
```

Comment:
in the eventing mechanism you can specify a event source. Maybe in future some other components want to create RuleEvents as well. So you can distinguish the events by the sources later...